src/main/resources/templates/admin/dashboard.html

```
@@ -521,6 +713,96 @@ <h2>RAW 상품 관리</h2>
+                        </div>
+                        <div class="full-span">
+                            <label for="bannerImageFile" class="form-label">배너 이미지 업로드</label>
+                            <div class="banner-image-upload-panel">
+                                <input type="file" class="form-control" id="bannerImageFile" accept="image/*">
+                                <input type="hidden" id="bannerImageUrl">
+                                <div class="banner-image-upload-actions">
+                                    <button type="button" id="upload-banner-image-button" class="btn btn-outline-primary">S3 업로드</button>
+                                    <span id="bannerImageUploadStatus" class="text-muted small">이미지를 선택하고 업로드하세요.</span>
+                                </div>
+                                <div id="bannerImagePreview" class="banner-image-preview">
+                                    <img id="bannerImagePreviewImg" src="" alt="배너 미리보기">
```

HTMLHint (1.9.2)

[error] The attribute [ src ] of the tag [ img ] must have a value.

(src-not-empty)